app/views/admin/application/_flashes.html.erb

```diff
@@ -0,0 +1,21 @@
+<%#
+# Flash Partial
+
+This partial renders flash messages on every page.
+
+## Relevant Helpers:
+
+- `flash`:
+  Returns a hash,
+  where the keys are the type of flash (alert, error, notice, etc)
+  and the values are the message to be displayed.
+%>
+
+<% if flash.any? %>
+  <div class="flashes">
+    <% flash.each do |key, value| -%>
+      <% next unless value.respond_to?(:html_safe) %>
+      <div class="flash flash-<%= key %>"><%= value.html_safe %></div>
+    <% end -%>
+  </div>
+<% end %>
```

Comment:
use flash for all the controllers not only for admin... Check what a flash message is